std/internal/math/errorfunction.d

```diff
@@ -16,7 +16,7 @@
  *  POWER = $1<sup>$2</sup>
  *  BIGSUM = $(BIG &Sigma; <sup>$2</sup><sub>$(SMALL $1)</sub>)
  *  CHOOSE = $(BIG &#40;) <sup>$(SMALL $1)</sup><sub>$(SMALL $2)</sub> $(BIG &#41;)
- *  TABLE_SV = <table border=1 cellpadding=4 cellspacing=0>
+ *  TABLE_SV = <table border="1" cellpadding="4" cellspacing="0">
```

Comment:
http://www.webdirections.org/blog/five-reasons-why-you-should-quote-attribute-values-in-html5/

I need these changes merged so we can have automatic HTML validation for PRs.